.checkstyle/checkstyle.xml

```diff
@@ -90,7 +90,7 @@
         <module name="MethodLength"/>
         <module name="ParameterNumber">
             <!-- default is 8 -->
-            <property name="max" value="13"/>
+            <property name="max" value="15"/>
```

Comment:
Isn't it better to suppress the warning with annotation on the class where you really need it instead of bumping the limit for all classes here?

Reply:
I can also do it like that yes.